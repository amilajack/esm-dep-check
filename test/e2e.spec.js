const edc = require('../src/DepCheck').default;

describe('e2e', () => {
  it('should test basic snapshot', () => {
    expect(edc()).toMatchSnapshot();
  });
});
