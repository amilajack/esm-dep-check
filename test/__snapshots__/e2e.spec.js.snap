// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`e2e should test basic snapshot 1`] = `
Array [
  Object {
    "name": "chalk",
    "supportsEsm": false,
  },
  Object {
    "name": "cli-table2",
    "supportsEsm": false,
  },
  Object {
    "name": "commander",
    "supportsEsm": false,
  },
]
`;
